docs/src/pages/components/data-grid/components/components.md

```diff
@@ -11,7 +11,8 @@ title: Data Grid - Components
 As part of the customization API, the grid allows you to override internal components with the `components` prop.
 The prop accepts an object of type [`GridSlotsComponent`](/api/data-grid/data-grid/#slots).
 
-If you wish to pass additional props in a component slot, you can do it using the `componentsProps` prop. This prop is of type `GridSlotsComponentsProps`.
+If you wish to pass additional props in a component slot, you can do it using the `componentsProps` prop.
+This prop is of type `GridSlotsComponentsProps`.
```

Comment:
This interface would be very interesting to add to the API documentation in the future
The comments in it are important to know each key impact on the UI.